File: Python/백준/Bronze/10988. 팰린드롬인지 확인하기/팰린드롬인지 확인하기.py
word = input().strip()

# 팰린드롬 확인: 문자열과 뒤집은 문자열이 같은지 비교
if word == word[::-1]:
    print(1)  # 팰린드롬이면 1
else:
    print(0)  # 팰린드롬이 아니면 0

File: Java/백준/Bronze/10988. 팰린드롬인지 확인하기/팰린드롬인지 확인하기.java
import java.util.Scanner;

public class Main {
    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);
        String word = sc.next();
        
        // 팰린드롬 확인
        boolean isPalindrome = true;
        int len = word.length();
        
        for (int i = 0; i < len / 2; i++) {
            if (word.charAt(i) != word.charAt(len - 1 - i)) {
                isPalindrome = false;
                break;
            }
        }
        
        // 결과 출력
        if (isPalindrome) {
            System.out.println(1);  // 팰린드롬이면 1
        } else {
            System.out.println(0);  // 팰린드롬이 아니면 0
        }
        
        sc.close();
    }
}